src/core_scan/Core_scan.ml

```diff
@@ -179,14 +179,14 @@ let sort_code_targets_by_decreasing_size (targets : Target_location.code list) :
   targets
   |> List_.sort_by_key
        (fun (target : Target_location.code) -> UFile.filesize target.file)
-       Int.compare
+       (Fun.flip Int.compare)
```

Comment:
lol. We should add a comment like "We want these in *descending* order"

Reply:
Sure.

Reply:
Done in c6fb06e5107f09a5caca7a57a18e5421aacb9f58